src/components/pill/pill-classic.style.js

```diff
@@ -0,0 +1,151 @@
+import { css } from 'styled-components';
+
+export const classicStyleConfig = {
+  disabled: {
+    color: '#CCD6DA',
```

Comment:
Where the colour is duplicated iyou could probably get away with just defining it the once rather than having `color` `bgColor` for example